File: test/SoundStream/test03.php
#!/usr/bin/php -c.

<? 

$srcdir = $argv[1];

$m = new SWFMovie(7);

$stream = new SWFSoundStream($srcdir . "/../Media/audio01.mp3");
$stream->setInitialMp3Delay(16384);

$mc = new SWFMovieClip();
$mc->setSoundStream($stream, $m->getRate());
for($i =  0; $i < 200; $i++)
	$mc->nextFrame();

$m->add($mc);
$m->nextFrame();

$m->save("test03.swf");

?>

File: test/SoundStream/test03.py
#!/usr/bin/python
from ming import *

import sys
mediadir = sys.argv[1]+'/../Media'

Ming_useSWFVersion(7);
m =  SWFMovie();

mc = SWFMovieClip(); 

stream = SWFSoundStream(mediadir + '/audio01.mp3')
stream.setInitialMp3Delay(16384);

mc.setSoundStream(stream, m.getRate())
for i in range (0, 200):
	mc.nextFrame()



m.add(mc);

m.save("test03.swf");
